### PR TITLE
Fix ios_command regex failure and ios_user shell timeout issue

### PR DESCRIPTION
**Pull request by ganeshrn**
##### SUMMARY

*  Increase logging buffer size on device
   before issuing logs that matches error regex
* increase shell timeout to avoid ios_user failure

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

File: tests/integration/targets/ios_user/tests/cli/auth.yaml [hunk 1]
```diff
@@ -60,21 +60,29 @@
...
+
+    - name: test with {{ ansible_user }} user without keys
+      cisco.ios.ios_command:
+        commands:
+          - show version
       ignore_errors: true
       register: results
+      vars:
+        ansible_user: ssh_user
+        ansible_private_key_file: ""
```
Comment (pabelanger) [inline tests/integration/targets/ios_user/tests/cli/auth.yaml:85]: Line 92 below, we first need to reset_connection now, as we are no longer using the zuul user.

Comment (ganeshrn) [reply tests/integration/targets/ios_user/tests/cli/auth.yaml:85]: Fixed

**Pull request merged**